lib/screens_web/views/v2/audio/shuttle_bus_info_view.ex

```diff
@@ -0,0 +1,10 @@
+defmodule ScreensWeb.V2.Audio.ShuttleBusInfoView do
+  use ScreensWeb, :view
+
+  def render("_widget.ssml", %{
+        minutes_range_to_destination: minutes_range_to_destination,
+        destination: destination
+      }) do
+    ~E|<p>Estimated <%= minutes_range_to_destination %> minute to <%= destination %></p>|
```

Comment:
This looks fine for now since we're expecting to get specific readout text from design soon. Though I think "minute" should be "minutes", e.g. "15-20 minutes".

Reply:
Right, this was more of a placeholder until we know exactly what we want to be read out for this widget. I knew that this line would be part of the readout, but I anticipate there being some additional text to add to it.